src/System.Net.WebSockets.WebSocketProtocol/tests/WebSocketProtocolTests.cs

```diff
@@ -108,6 +101,38 @@ public void CreateFromStream_ValidBufferSizes_Succeed(int bufferSize)
             }
         }
 
+        [Fact]
+        public static async Task ManagedWebSocket_ReceiveUTF8SplitAcrossMultipleBuffers()
+        {
+            // 1 character - 2 bytes
+            var payload = Encoding.UTF8.GetBytes("�");
```

Comment:
Please prefer \uNNNN syntax for non ascii characters  so that we don't lose your test character later through someone mistakenly changing file encoding.